pkg/be/kubernetes/shell/chart/templates/blob.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.blob.cm.data }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ print (.Release.Name | trunc 44) "-blob" }}
```

Comment:
i think the idea of `trunc 44` is that it should be 53 minus len("-blob") so `trunc 48` i guess?